pkg/deploy/controller/generictrigger/controller.go

```diff
@@ -194,6 +194,12 @@ func (c *DeploymentTriggerController) update(config *deployapi.DeploymentConfig,
 	config.Status.LatestVersion++
 	config.Status.Details = new(deployapi.DeploymentDetails)
 	config.Status.Details.Causes = causes
+	switch causes[0].Type {
+	case deployapi.DeploymentTriggerOnConfigChange:
+		config.Status.Details.Message = "caused by a config change"
+	case deployapi.DeploymentTriggerOnImageChange:
+		config.Status.Details.Message = "caused by an image change"
```

Comment:
Messages *probably* should be sentences (or be capitalized).  It's fine for now but we should revisit